### DISCUSSION TITLE
Would it be useful to ask for validation to quit when there is more than 1 buffer opened?

### DISCUSSION THREAD

**Discussion by jerabaul29** [helix-editor/helix#9250] [Ideas]
I come from vim and I am used to a "tabs" way of thinking, where ```:q``` closes the current tab, not the whole vim program. But in helix, ```:q``` closes the view (ie the whole helix program), not just the current buffer. Due to muscle memory, I have quite a few time ended up closing the view / helix program / a large number of buffers, while I only wanted to close the current buffer. I was chatting with a friend and he was confused at the start by the same issue.

Hence my question: when there is more than 1 buffer opened (which in vim would be "several tabs"), and the author presses ```:q```, would it make sense to have a popup showing up:

"
You have several buffers open, and :q will close them all and quit helix.
Are you sure you want to quit? [y: want to quit; any other: abort the :q command]
"

or something in this kind? :)

**Comment by seungjin** [comment]
Yep, keep pressing :q!, not just 'buffer-close'.
When I use Emacs, there's an option that shows a confirmation message for C-c C-x (to exit Emacs).
The same option would be very helpful here!
